packages/api/src/tasks/index.ts

```diff
@@ -70,15 +71,18 @@ if (!RUN_SCHEDULED_TASKS_EXTERNALLY) {
   });
 } else {
   logger.warn('In-app cron job is disabled');
-  main(taskName)
-    .then(() => {
-      process.exit(0);
-    })
-    .catch(err => {
-      console.log(err);
-      logger.error(serializeError(err));
-      process.exit(1);
-    });
+  if (manual) {
```

Comment:
adding the extra flag to prevent the task from running if `RUN_SCHEDULED_TASKS_EXTERNALLY` is true